### PR TITLE
@dzucconi: Generic error modal component

### PR DESCRIPTION
**Pull request by craigspaeth**
This adds a generic error alert modal that can easily be passed as an error handler to catch unexpected errors. The large diff is mostly the addition of node-mandrill to allow reporting functionality (will need this addon later anyways for things like email notifications to Editorial when a partner publishes an article).

The motive for putting this in now is to at-least provide feedback when "Sync to Post" fails.

cap

### CODE REVIEW

File: client/components/error_modal/index.coffee [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <br>
+        <b>Error message:</b>
+        <pre>#{@error.message}</pre>
+        <br>
+        <b>Stack trace:</b>
+        <pre>#{@error.stack}</pre>
+        """
+    flash
+      message: @flashMessage or "Thanks for your bug report."
+      timeout: 2000
```
Comment (dzucconi) [inline client/components/error_modal/index.coffee]: V good idea

File: .env.example [hunk 2]
```diff
@@ -8,7 +8,9 @@ ARTSY_URL=https://stagingapi.artsy.net
 SESSION_SECRET=p0s1tr0n
 MONGOHQ_URL=mongodb://localhost:27017/positron
 GRAVITY_CLOUDFRONT_URL=https://d1ycxz9plii3tb.cloudfront.net
+TECH_SUPPORT=craig@artsymail.com
```
Comment (dzucconi) [inline .env.example]: :neckbeard:

**Comment by dzucconi** [comment]
There no CI for Positron yet? Or did the hook just not grab...

**Comment by craigspaeth** [comment]
Yeah I was thrown off by that... the hook just didn't grab. Must be the dust that settled on this repo :-P Hopefully the next PR kicks it.

**Comment by dzucconi** [comment]
Ah woops; my bad :\

**Comment by craigspaeth** [comment]
No worries, should have been green anyways right.... "Works on my machine" :grin:

**Pull request merged**